pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1007,6 +1008,17 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		}
 	}
 
+	// Append HostDev to libvirt DomXML if QAT is required
+	if util.IsQATVMI(vmi) {
```

Comment:
This looks like at the end we only do ordinary device passthrough ...

Maybe we could test it with a mock pci device which gets added to the pod instead of the QAT device? This way we could in the future test all pic passthrough devices.

What do others think?

Reply:
Yes, It is an ordinary device passthrough: get the QAT vf device id and add it to libvirt domain as a hostdev.